roles/scale_object/cluster/meta/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+galaxy_info:
+  role_name: obj_cluster
+  author: IBM Corporation
+  description: Highly-customizable Ansible role for installing and configuring IBM Spectrum Scale (GPFS)
```

Comment:
I can create a separate issue to add this description to make it object protocol.